File: bm_json_xam_platform/Shared/Model/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Shared.Model
{
    class Person
    {
        public string Country { get; set; }
        public string Birthday { get; set; }
        public string Phonenumber { get; set; }
        public Adress Location { get; set; }
        public string Surname { get; set; }
        public string Forename { get; set; }

        public Person()
        {
            Country = "Germany";
            Birthday = "13.08.1985";
            Phonenumber = "+490154868686";
            Location = new Adress();
            Surname = "Mustermann";
            Forename = "Max";
        }
    }
}

File: bm_json_android/app/src/main/java/steffen/de/bm_json_android/Model/Person.kt
package steffen.de.bm_json_android.Model

import android.R.attr.name



data class Person (    val country: String = "Germany",
                       val birthday: String = "13.08.1985",
                       val phonenumber: String = "+490154868686",
                       val location: Adress = Adress(),
                       val surname: String = "Mustermann",
                       val forename: String = "Max"){
    override fun toString(): String {

        return String
                .format("Person: [Country: %s, Birthday: %s, Phonenumber: %s, Location: %s, Surname: %s, Forename: %s]",
                        country, birthday, phonenumber, location, surname, forename)
    }
}